astropy/io/ascii/latex.py

```diff
@@ -259,10 +270,19 @@ class Latex(core.BaseReader):
             names is added, which contains the values of the
             dictionary. Example::
 
-              from astropy.io import ascii
-              data = {'name': ['bike', 'car'], 'mass': [75,1200], 'speed': [10, 130]}
-              ascii.write(data, Writer=ascii.Latex,
-                               latexdict = {'units': {'mass': 'kg', 'speed': 'km/h'}})
+              >>> from astropy.io import ascii
```

Comment:
Same comment, showing the output here makes it harder to read the documentation.